core/solidus_core.gemspec

```diff
@@ -35,9 +35,9 @@ Gem::Specification.new do |s|
   s.add_dependency 'discard', '~> 1.0'
   s.add_dependency 'friendly_id', '~> 5.0'
   s.add_dependency 'kaminari-activerecord', '~> 1.1'
-  s.add_dependency 'monetize', '~> 1.1'
+  s.add_dependency 'monetize', '~> 1.8.0'
```

Comment:
think this could be `~> 1.8` since it works with both 1.8 and 1.9 versions

Reply:
you are right, fixed!